Spring-04

Max-Pochepets opened the pull request:
Configured Spring MVC

Max-Pochepets committed:
Initial

Max-Pochepets committed:
first

Max-Pochepets committed:
Initial

Max-Pochepets committed:
Refactor:

changed format of MovieSessionResponseMapper's show time
created a Postman requests collection

Max-Pochepets committed:
Refactor:

changed get() in CinemaHallService and MovieService

Max-Pochepets committed:
Refactor:

get() in CinemaHall and Movie DAO's simplified

Mandybur800 commented on src/main/java/cinema/shop/config/AppConfig.java:
```diff
@@ -0,0 +1,51 @@
+@ComponentScan(basePackages = {
+        "cinema.shop.dao",
+        "cinema.shop.service",
+        "cinema.shop.model",
```
why do you add models to beans?

Max-Pochepets replied:
My bad, thank you!

Max-Pochepets committed:
removed model from packages being scanned

Max-Pochepets committed:
removed model from packages being scanned

Max-Pochepets committed:
Fix:

added Id field to MovieSessionRequestDto

Max-Pochepets committed:
Fix:

removed Id from MovieSessionRequestDto
added path variables to MovieSessionController's update and delete
changed the exception being thrown from getting object out of Optional from Runtime to NoSuchElement

Max-Pochepets committed:
checkstyle

Max-Pochepets committed:
Fix:

changed MovieSessionController logic
chain of changes goes down until the DAO layer

Max-Pochepets committed:
Merge pull request #7 from Max-Pochepets/Spring-05

Spring-05
